docs_src/getting-started/Ch-GettingStartedSnapUsers.md

```diff
@@ -105,50 +101,79 @@ Mapping examples:
-
-For example, to change the service port of the core-data service on `edgexfoundry` snap to 8080:
-```bash
-snap set edgexfoundry app-options=true
-snap set edgexfoundry apps.core-data.config.service-port=8080
-```
+!!! Example 
+    To change the service port of the `core-data` service on `edgexfoundry` snap to 8080:
+    ```bash
+    snap set edgexfoundry apps.core-data.config.service-port=8080
+    ```
+    This would internally export `SERVICE_PORT=8080` to `core-data` service.
 
-The services load the set config options on startup. If the service has already started, a restart is necessary to load them.
+!!! Note
+    The services load the set config options on startup. If a service has already started, a restart will be necessary to load the configurations.
 
-#### Common configuration examples
+#### Examples
 ##### Disabling security
 [disabling security]: #disabling-security
-
+    
 !!! Warning
     Disabling security is NOT recommended, unless for demonstration purposes, or when there are other means to secure the services.
 
-    The snap will NOT allow the Secret Store to be re-enabled. The only way to re-enable the Secret Store is to re-install the snap.
-    
-The Secret Store is used by EdgeX for secret management (e.g. certificates, keys, passwords). Use of the Secret Store by all services can be disabled globally. Note that doing so will also disable the API Gateway, as it depends on the Secret Store.
+    The [platform snap] snap does NOT allow the security to be re-enabled. The only way to re-enable it is to re-install the snap.
+
+Disabling security involves two steps:
 
-The following command disables the Secret Store and in turn the API Gateway:
+1. Stopping the security services and disabling them so that they don't run again.
+2. Configuring EdgeX services to NOT use the Secret Store by setting [EDGEX_SECURITY_SECRET_STORE](../../microservices/configuration/CommonEnvironmentVariables/#edgex_security_secret_store) to false.
+
+The [platform snap] includes all the reference security components. The security components are enabled by default. The platform snap provides a convenience way to perform both of the above by setting `security=false`:
 ```bash
-sudo snap set edgexfoundry security-secret-store=off
+snap set edgexfoundry security=false
 ```
 
-All services in the snap except for the API Gateway are restricted by default to listening on localhost (127.0.0.1).
-The API Gateway proxies external requests to internal services.
-Since disabling the Secret Store also disables the API Gateway, the service endpoint will no longer be accessible from other systems.
-They will be still accessible on the local machine for demonstration and testing.
+The above command results in stopping and disabling the security components as well as setting `config.edgex-security-secret-store=false` internally so that core services stop using the Secret Store. The services are automatically restarted to pick up the new configuration.
```

Comment:
How about moving these as an example section right after step 1?

Reply:
This covers both step 1 and step 2, as explained in the text.

Reply:
I see, I wasn't aware that the "EdgeX services" here include core services.